### ISSUE TITLE
V2.8.0 started to explain the code

### ISSUE THREAD

**Issue by vadviktor** [Blarc/ai-commits-intellij-plugin#285]
Hey @Blarc ,

I started using your plugin a few days ago. It was working fine with Ollama. Today, after updating to 2.8.0, it started explaining the code in the commit diff rather than generating a message for it.

Is there a way to debug the input and find out what the issue is on my end? Could you please help?

**Comment by Blarc** [comment]
Hey @vadviktor,

that seems odd, because I didn't change any of the logic for generating commit messages. I'd suggest you check the prompt that is sent to the model by opening the edit prompt dialog in the settings and checking the prompt preview.

You could also try testing with some other models, to see if the model you're using is the problem.

**Comment by vadviktor** [comment]
Thanks for your reply @Blarc !

Sadly, I do not remember what I did or didn't do yesterday to find a solution to this issue I was seeing, but today, it does not happen. Could it be that I've forgotten to restart Rider? 🤔

I do remember that I was swapping around, testing different models through Ollama.

❓ 
This reminds me, and since we are in the discussion: why can I add multiple models to this list? I have not found any way to select from these nor any documentation around this list. Could you tell me more about it, please? 🙏 

![rider64_0xwzRBXoy1](https://github.com/user-attachments/assets/ad86c553-29b2-4764-a77d-ed769d4d4550)

I now thought that having multiple models in this list did something to the plugin that produced the buggy behaviour, but testing it today didn't break it. I've added multiple, and I even restarted Rider, and it works fine. However, I can only tell which model is answering because I look at Ollama's cli to see which model is loaded.

I'm sorry for the noise 😓. I'll give it another day and a few more tests before reporting any bugs in the future.

**Comment by vadviktor** [comment]
🤦 I've answered my question above, sorry. I figured that because I didn't set a different name to the model setups, I only saw Ollama. I saw the three same-named entries when I checked the "LLM Client" dropdown. 😅 

Wouldn't it improve the UX if you didn't allow any models to be named the same? This is just a suggestion.

Cheers! I'm closing this ticket because there is nothing to resolve here. Thanks for your time!
